### ISSUE TITLE
phpcs: Referenced sniff "FreshConsulting.WCAG20.Violations" does not exist

### ISSUE THREAD

**Issue by netdjw** [FreshConsulting/FreshConsulting-WCAG-Linter#2]
I use the PHP_CodeSniffer version 3.5.4 (stable), cloned the repository to a directory, ran the `sudo phpcs --config-set installed_paths /path/to/wcag` (btw upgrade the docs, add `sudo` to the command please) with this re```sult:

```
Using config file: /etc/php-codesniffer/CodeSniffer.conf

Config value "installed_paths" added successfully
```

Then restarted VSCode and get this error message:
```
phpcs: Referenced sniff "FreshConsulting.WCAG20.Violations" does not exist
```

Then I tried to run this command in console:
```
phpcs --sniffs=FreshConsulting.WCAG20.Violations ./
```
And get this result:
```
ERROR: No sniffs were registered

Run "phpcs --help" for usage information
```

Any idea what I missed?

Thanks!

=== UPDATE ===

```
$ cat /etc/php-codesniffer/CodeSniffer.conf
<?php
 $phpCodeSnifferConfig = array (
  'default_standard' => 'PEAR',
  'php_path' => '/usr/bin/php',
  'installed_paths' => '/path/to/wcag',
)
```
I think here is something missing.

=== UPDATE 2 ===

After I realized I made a mistake with the path of the cloned repository, I fixed it in the `/etc/php-codesniffer/CodeSniffer.conf` file to this: `/home/djw/tools/development/wcag` then `/home/djw/tools/development/wcag/`, then `/home/djw/tools/development/wcag/FreshConsulting` but none worked.
